packages/cli/src/commands/deploy/args.ts

```diff
@@ -41,7 +42,7 @@ export const help = () => {
   ${chalk.gray('–')} Deploy with prebuilt outputs
 
     ${chalk.cyan(`$ ${getPkgName()} build`)}
-    ${chalk.cyan(`$ ${getPkgName()} deploy --prebuilt`)}
+    ${chalk.cyan(`$ ${getPkgName()} deploy --skip-build`)}
```

Comment:
I think `vercel deploy --skip-build` is confusing.

You're not telling it to skip the build, you're telling it that you are deploying files you already built which is why I think `--prebuilt` makes more sense.

Reply:
I think it depends on how we frame it for the user. The option `--skip-build` makes sense when you are thinking about deployments steps: "skip the build because I already did it".

If we did allow a different path argument for where to find the BOA, then yes, the `--skip-build` makes less sense. But we could rearrange that:

```
vercel deploy ./path/to/output --skip-build
```

In this framing, we're deploying the built output. You could say we don't even need the `--skip-build` option in this case, but I think being explicit here is helpful. Technically you could want to deploy something that looks like BOA, but as static assets.